scripts/local-setup/arch.sh

```diff
@@ -0,0 +1,170 @@
+
+    git@github.com:ONSdigital/dp-dataset-api
+    git@github.com:ONSdigital/dp-frontend-filter-dataset-controller
+    git@github.com:ONSdigital/dp-frontend-geography-controller
+
+    git@github.com:ONSdigital/dp-recipe-api
+    git@github.com:ONSdigital/dp-import-api
+    git@github.com:ONSdigital/dp-upload-service
+    git@github.com:ONSdigital/dp-import-tracker
+    git@github.com:ONSdigital/dp-dimension-extractor
+    git@github.com:ONSdigital/dp-dimension-importer
+    git@github.com:ONSdigital/dp-observation-extractor
+    git@github.com:ONSdigital/dp-observation-importer
+    git@github.com:ONSdigital/dp-hierarchy-builder
+    git@github.com:ONSdigital/dp-dimension-search-builder
+    git@github.com:ONSdigital/dp-publishing-dataset-controller
+
+    git@github.com:ONSdigital/dp-cantabular-server
+    git@github.com:ONSdigital/dp-cantabular-api-ext
+    git@github.com:ONSdigital/zebedee
+    git@github.com:ONSdigital/dp-recipe-api
+    git@github.com:ONSdigital/dp-import-api
+    git@github.com:ONSdigital/dp-dataset-api
+    git@github.com:ONSdigital/dp-import-cantabular-dataset
+    git@github.com:ONSdigital/dp-import-cantabular-dimension-options
+
+    git@github.com:ONSdigital/dp-dimension-search-api
+    git@github.com:ONSdigital/dp-code-list-api
+    git@github.com:ONSdigital/dp-hierarchy-api
+    git@github.com:ONSdigital/dp-filter-api
+    git@github.com:ONSdigital/dp-dataset-exporter
+    git@github.com:ONSdigital/dp-dataset-exporter-xlsx
+
+    git@github.com:ONSdigital/dp-zebedee-content.git
+)
+
+for repo in ${arr[@]}
+do
+  (
+    cd ../../..
+    path=$(echo $repo | cut -d "/" -f 2)
+    if [[ -d "$path" ]]; then
+        echo "path: $path already exists... skipping"
+    else
+      set +e
+      git clone "$repo"
+      set -e
+    fi
+  )
+done 
+
+(cd ../../../dp-cli/cmd/dp; go install)
+(cd ../../../dp-compose; docker compose up --wait)
+(cd ../../../dp-zebedee-content; make install)
+(cd ../../../dp-cli; cp -a config/example_config.yml ~/.dp-cli-config.yml)
+(cd ../../../dp-cli; cp -a config/example_config.yml ~/.dp-cli-config.yml)
+(cd ../../../dp-ci; git checkout main; git pull)
+(cd ../../../dp-setup; git checkout main; git pull)
+
+working_dir=$(cd ../../..; pwd)
+
+VAR=$(cat <<EOF
+dp-setup-path: "$working_dir/dp-setup" # The path to the dp-setup repo on your machine.
+dp-ci-path: "$working_dir/dp-ci" # The path to the dp-ci repo on your machine.
+dp-hierarchy-builder-path: "$working_dir/dp-hierarchy-builder" # The path to the dp-hierarchy-builder repo on your machine.
+dp-code-list-scripts-path: "$working_dir/dp-code-list-scripts" # The path to the dp-code-list-scripts repo on your machine.
+EOF
+)
+
+echo "$VAR 
+$(tail +8 ~/.dp-cli-config.yml)" > ~/.dp-cli-config.yml
+sed -i "s/ssh-user: JamesHetfield/ssh-user: ${SSHUSER}/" ~/.dp-cli-config.yml
+
+dp remote allow sandbox
+
+STARTUP_FILE=$(cat <<'EOF'
+# Digital Publishing services
+export zebedee_root=~/Documents/website/zebedee-content/generated
```

Comment:
I would probably store this outside Documents directory
```suggestion
export zebedee_root=~/ons/website/zebedee-content/generated
```